lib/commons/dom/is-visible.js

```diff
@@ -33,7 +33,8 @@ function isClipped(clip) {
  */
 dom.isVisible = function(el, screenReader, recursed) {
 	'use strict';
-	var style, nodeName, parent;
+	const node = axe.utils.getNodeFromTree(el);
+	const cacheName = 'isVisible' + (screenReader ? 'ScreenReader' : '');
```

Comment:
Given that this isn't a variable that will always be available, lets make this `_isVisible`.